creation/web_base/get_id_selectors.source

```diff
@@ -3,22 +3,24 @@
 # SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
 # SPDX-License-Identifier: Apache-2.0
 
+# NOTE: any script that imports this script to use the functions defined must also import add_config_line.source
```

Comment:
Make sure that this is currently true. 
If you verified already ignore this comment